package.json

```diff
@@ -84,10 +88,14 @@
     "react-feather": "^2.0.9",
     "react-ga": "^3.3.0",
     "react-is": "^17.0.2",
+    "react-markdown": "^8.0.0",
     "react-portal": "^4.2.1",
+    "react-query": "^3.34.16",
     "react-redux": "^7.2.0",
     "react-router-dom": "^6.2.2",
+    "react-slick": "^0.28.1",
```

Comment:
we also need to install `slick-carousel` as you have removed from interface.

Reply:
@jb1011 its not like that. this components will be used by others also. so we want other to install as less as possible. so its good to install `slick-carousel` if possible